src/cryptography/hazmat/bindings/_rust/declarative_asn1.pyi

```diff
@@ -20,9 +20,20 @@ class Type:
     PyStr: typing.ClassVar[type]
 
 class Annotation:
+    default: Default | None
     def __new__(
         cls,
+        default: Default | None = None,
```

Comment:
I guess my question is: what's the argument for this being a `Default` instance here vs. just a `typing.Any`?